### PR TITLE
Fix Morning and Zenburn themes on mobile

### PR DESCRIPTION
**Pull request by floogulinc**
See Shuo-IRC/Shuo#46 for the original issue (fixed in Shuo-IRC/Shuo#85)

Before
--
The connect and settings buttons are invisible.

Morning | Zenburn
---- | ----
screenshot_2015-08-09-06-27-33 | screenshot_2015-08-09-06-27-53

After
--
Morning | Zenburn
---- | ----
screenshot_2015-10-03-20-39-50 | screenshot_2015-10-03-20-39-58

### CODE REVIEW

File: client/themes/zenburn.css [hunk 1]
```diff
@@ -18,6 +18,10 @@ BORDERS #333333
 QUIT #bc6c4c
 */
 
+body {
+	background: #2b2b2b;
+}
+
```
Comment (JocelynDelalande) [inline client/themes/zenburn.css]: Is that really related (or needed) ?

Comment (floogulinc) [inline client/themes/zenburn.css]: @JocelynDelalande Yes it is, this is what it looks like without that:

[screenshot_2015-10-07-05-14-22](https://cloud.githubusercontent.com/assets/1300395/10334821/60283aa0-6cb2-11e5-8564-4059b9cf26ec.png)

**Comment by JocelynDelalande** [comment]
@floogulinc thanks for your attention on details :)

But I don't see, what's wrong ?

**Comment by floogulinc** [comment]
@JocelynDelalande as you can see, without the background set, it leaks the default theme's background color in that one spot. Morning doesn't need this because it uses the same color as the default.

**Comment by JocelynDelalande** [comment]
ok :+1: then :)

**Comment by astorije** [comment]
:+1:

**Comment by astorije** [comment]
Once more, thanks @floogulinc!!
